chef/cookbooks/crowbar/templates/default/crowbar-upgrade-os.sh.erb

```diff
@@ -67,13 +67,12 @@ initiate_node_upgrade()
 
     trap cleanup INT EXIT
 
-    # we need to change the horizon django NullHandler in the custom conf files as its been
-    # deprecated so horizon related packages wont be able to install correctly, as they run some
-    # manage.py commands, which loads the conf files and errors out
-    # only needed from soc7 -> soc8
-    horizon_custom_config="/srv/www/openstack-dashboard/openstack_dashboard/local/local_settings.d/_100_local_settings.py"
-    if [[ -f $horizon_custom_config ]] ; then
-      sed -i s/django.utils.log.NullHandler/logging.NullHandler/g $horizon_custom_config
+    update_dashboard_theme_SUSE=0
```

Comment:
while this may fix the issue with the pdf thingie (which should be solved in the package IMO) the old problem with custom conf files having deprecated params still exists, so I would recommend keeping the old change and just adding this extra part.

Reply:
Good point